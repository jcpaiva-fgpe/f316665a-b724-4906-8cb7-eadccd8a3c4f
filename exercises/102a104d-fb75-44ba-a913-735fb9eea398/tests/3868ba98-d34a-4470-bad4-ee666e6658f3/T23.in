16{x}+([e]+52w)+12[1u-(62b+5y)+28{9{0o}-f}+[s]+(3[50((88([q]-39p)))])+65{[89(14c)]}-(95y-48b)]+{[91(26(17p))]+55z-20{[9{[89{g}]-v-68(15(a)+29[[94(f)]])-70[({71b})]}]-81(66[{d+(79{73a})-[82m]}+o])}-{53({25k}-19(94({2f+x}+e)+27(44{(67w+90{b}+{62[k]}-z)+24l+z-30a})))}+([p-43x-37{(11s)-m+36g}])-{53{p}}}+(18j-78f-27h)
