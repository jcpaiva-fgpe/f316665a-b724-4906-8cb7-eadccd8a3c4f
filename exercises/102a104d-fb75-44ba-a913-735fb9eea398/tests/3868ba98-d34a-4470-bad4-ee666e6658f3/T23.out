{} 2 4
[] 7 9
() 6 14
() 22 29
{} 35 38
{} 33 41
[] 43 45
[] 57 59
() 56 64
() 53 65
() 52 66
[] 49 67
() 47 68
() 76 80
[] 73 81
{} 72 82
() 84 92
[] 18 93
() 102 106
() 99 107
[] 96 108
{} 123 125
[] 120 126
() 135 137
() 145 147
[] 142 148
[] 141 149
() 132 150
{} 156 160
() 155 161
[] 154 162
{} 119 163
[] 117 164
{} 178 182
() 175 183
[] 185 189
{} 172 190
[] 171 193
() 168 194
{} 116 195
{} 201 205
{} 213 218
() 212 221
{} 236 238
[] 243 245
{} 240 246
() 229 249
{} 228 260
() 225 261
() 209 262
() 200 263
{} 197 264
() 277 281
{} 276 288
[] 267 289
() 266 290
{} 295 297
{} 292 298
{} 95 299
() 301 313
